app.json

```diff
@@ -6,7 +6,7 @@
     "orientation": "portrait",
     "icon": "./assets/images/icon.png",
     "scheme": "myapp",
-    "userInterfaceStyle": "automatic",
+    "userInterfaceStyle": "light",
```

Comment:
Are we just assuming/forcing everyone to use light mode here?

Reply:
Why yes we are. Dark mode is for suckers (even though I am a big fan of dark mode but besides the point)